client/src/helpers/APICalls/getReviews.ts

```diff
@@ -0,0 +1,4 @@
+export const getReviews = async (sitterId: string) => {
+  const response = await fetch(`http://localhost:3000/reviews/all/${sitterId}`);
```

Comment:
Nit: Can we please specify the options for the fetch client (please see handler above)